### PR TITLE
MAT-4756 fix measure transfer issue and update madie data model

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## PR Template

Jira Ticket: [MAT-4756](https://jira.cms.gov/browse/MAT-4756)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e. Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/utils/measureConversionUtils.ts [hunk 1]
```diff
@@ -143,7 +201,47 @@ export const convertToMadieMeasure = (matMeasure: MatMeasure): Measure => {
     cqlLibraryName: cqlLibraryName,
     createdBy: matMeasure.harpId,
     lastModifiedBy: matMeasure.harpId,
+    versionId: uuidv4(),
```
Comment (jkotanchik-SB) [inline lambda-function/src/utils/measureConversionUtils.ts:204]: Any reason for generating a new uuid versus using the measure.id value?

Comment (sb-cecilialiu) [reply lambda-function/src/utils/measureConversionUtils.ts:204]: @jkotanchik-SB No particular reason, just wanted to make sure it's there because it's a required field in Madie. I'll change it to measure id and remove uuid. Thanks Joe!

Comment (sb-cecilialiu) [reply lambda-function/src/utils/measureConversionUtils.ts]: @jkotanchik-SB Just pushed in the change thanks!

File: lambda-function/src/utils/measureConversionUtils.ts [hunk 2]
```diff
@@ -66,15 +74,23 @@ const convertPopulations = (matPopulations: any) => {
   const populations = matPopulations.reduce((populationMap: MadiePopulationType, population: any) => {
```
Comment (adongare) [inline lambda-function/src/utils/measureConversionUtils.ts:74]: I think this function is operating on single population all the time now. Can we rename is to `convertPopulation`? also do you think `reduce` is not required anymore?

Comment (sb-cecilialiu) [reply lambda-function/src/utils/measureConversionUtils.ts:74]: @adongare I was wondering about the use of reduce? Unless you need both previous population and current population?

File: lambda-function/src/utils/measureConversionUtils.ts [hunk 3]
```diff
@@ -87,12 +103,48 @@ const convertMeasureGroups = (measureResourceJson: string): Array<Group> => {
       scoring: measureResource.scoring.coding[0].display,
     } as Group;
 
-    const populations = convertPopulations(group.population);
-    madieMeasureGroup.population = populations;
+    //const population = convertPopulations(group.population);
+    const populations: Population[] = [];
+    let result = Object.entries(group.population);
+    result.map((item) => {
```
Comment (adongare) [inline lambda-function/src/utils/measureConversionUtils.ts:109]: if you don't want to return anything (seems like mapping is not happening) either you can use foreach/for of or modify mapping like:

```
populations = result.map((item) => {
      return convertPopulations(item[1]);
    }); 
```

Comment (sb-cecilialiu) [reply lambda-function/src/utils/measureConversionUtils.ts]: @adongare Ok did some work on this, just checked the code in, thanks!

**Review by jkotanchik-SB — APPROVED** [review]
